addons/controllers/calicoconfig_controller_test.go

```diff
@@ -21,34 +21,62 @@ import (
 
 const testCluster = "test-cluster-calico"
 
-var _ = Describe("CalicoConfig Reconciler", func() {
+var _ = Describe("CalicoConfig Reconciler and Webhooks", func() {
 	var (
-		clusterName             string
-		clusterResourceFilePath string
+		clusterName string
 	)
 
+	const (
+		clusterResourceFilePath = "testdata/test-calico.yaml"
+	)
 	JustBeforeEach(func() {
+		// Create the admission webhooks
```

Comment:
Code under `JustBeforeEach ` will be executed before each test case. It will complain resource already exist if there is more than one testcase. Consider move it somewhere else, in case we add more test cases in the future.

Reply:
Yes the resources are created in `JustBeforeEach` but they are also deleted in `AfterEach`